source/concurrent/native/ConditionWin32.ooc

```diff
@@ -45,7 +37,7 @@ ConditionWin32: class extends WaitCondition {
 		this _waitingEvents add(eventId)
 		this _mutex unlock()
 		mutex unlock()
-		(WaitForSingleObject(eventId, Infinite) == WaitSuccess)
+		(WaitForSingleObject(eventId, INFINITE) == WaitSuccess)
```

Comment:
I don't know. This was the only place it was called `Infinite`, all the others used `INFINITE`. @fredrikbryntesson , any opinion?